controllers/mysqlcluster_controller.go

```diff
@@ -260,6 +296,101 @@ func generateRandomBytes(n int) ([]byte, error) {
 	return ret, nil
 }
 
+func mergeMycnf(cnf1, cnf2 map[string]string) {
+	for k, v := range cnf2 {
+		cnf1[normalizeConfKey(k)] = v
+	}
+}
+
+func normalizeConfKey(k string) string {
+	return strings.ReplaceAll(k, "-", "_")
+}
+
+func (r *MySQLClusterReconciler) createOrUpdateConfigMap(ctx context.Context, log logr.Logger, cluster *mocov1alpha1.MySQLCluster) error {
+	cm := &corev1.ConfigMap{}
+	cm.SetNamespace(cluster.Namespace)
+	cm.SetName(cluster.Name)
+
+	op, err := ctrl.CreateOrUpdate(ctx, r.Client, cm, func() error {
+		cm.Labels = map[string]string{
```

Comment:
Are you sure to drop all existing labels?

Reply:
Yes, we're pretty sure, because Moco manages the cm, and there're no labels added other than that.

Reply:
OK